Doc/whatsnew/3.11.rst

```diff
@@ -1571,6 +1571,11 @@ New Features
 * Added the :c:member:`PyConfig.safe_path` member.
   (Contributed by Victor Stinner in :gh:`57684`.)
 
+* Added the new limited C API function :c:func:`PyType_FromMetaModuleAndSpec`,
+  which generalizes the existing :c:func:`PyType_FromModuleAndSpec` using
+  an additional metaclass argument.
+  (Contributed by Wenzel Jakob in :issue:`93012`.)
```

Comment:
```suggestion
  (Contributed by Wenzel Jakob in :gh:`93012`.)
```

Reply:
`:issue:` is for the old bug tracker (bugs.python.org)

Reply:
Got it, thanks!